bindings/Python/py11IO.cpp

```diff
@@ -111,10 +111,10 @@ Variable IO::InquireVariable(const std::string &name)
     helper::CheckForNullptr(m_IO, "for variable " + name +
                                       ", in call to IO::InquireVariable");
 
-    const std::string type(m_IO->InquireVariableType(name));
+    const Type type(m_IO->InquireVariableType(name));
     core::VariableBase *variable = nullptr;
 
-    if (type == "unknown")
+    if (type == Type::None)
```

Comment:
@bradking most places used an empty string for "no type", but a few used `"unknown"`.  I've converted both cases to `Type::None`.  Do you know why there was a distinction before?

Reply:
No idea.  @eisenhauer @pnorbert ?

Reply:
Only @williamfgc  might know, but I think unknown came from adios1 where this (invalid) type value existed. I don't remember if it was ever used there other than initializing structs. I assume this is safe to get rid of as there should not be any variable object existing with unknown type in adios2.

Reply:
The new `GetTypeFromString` conversion function that maps from the old string values (still used in the public API) to the new `adios2::Type` enumeration will convert any unknown string value to `Type::None`, so if we somehow get an `"unknown"` value it will become `Type::None` and match these conditions as before.

Reply:
@pnorbert is right, at some point we tried to keep it open for supporting POD structs, but it was always pushed in the list of priorities.

Reply:
Also, if I recall correctly the design decision at that time for using string was related to POD types as we're exploring ways of passing something generic/fixed like "struct" (POD) and something the user could set at runtime "Points3D".  Also, most string types are <= 15 characters and they are put on the stack for small string optimization (SSO). Using SSO was the recommendation from my previous employer as cost comparisons in the L-* cache stack in highly vectorized architectures are essentially undetectable. All good intentions, but POD types we not a priority in a HPC production library as there are added cost associated to offsets and memory alignment so we never went further. Hope it helps. Thanks for cleaning this.

Reply:
Okay, thanks for explaining!